src/NuGet.Clients/NuGetConsole.Host.PowerShell/PowerShellHost.cs

```diff
@@ -312,10 +314,24 @@ public void Initialize(IConsole console)
                         }
 
                         UpdateWorkingDirectory();
+
+                        if (!PowerShellLoaded)
+                        {
+                            var telemetryEvent = new TelemetryEvent(NuGetPowerShellUsageCollector.PowerShellLoaded, new Dictionary<string, object>
```

Comment:
Sometime earlier I left feedback to use classes for events instead of just a dictionary of properties. 
It's a pattern followed in https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/PackageSourceTelemetry.cs, https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/RestoreTelemetryEvent.cs etc.

Reply:
@nkolev92 
I created this [PR#3863](https://github.com/NuGet/NuGet.Client/pull/3863) to address this.